test/e2e/storage/volume_metrics.go

```diff
@@ -168,12 +168,8 @@ var _ = utils.SIGDescribe("[Serial] Volume metrics", func() {
 		framework.ExpectNoError(err, "Error getting default storageclass: %v", err)
 		pluginName := defaultClass.Provisioner
 
-		controllerMetrics, err := metricsGrabber.GrabFromControllerManager()
```

Comment:
Can you add a TODO to this test case referencing the bug tracking the provision metric followup issue?

Reply:
done